src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -15,7 +15,6 @@ module sonic-device_metadata {
 
     import sonic-types {
         prefix stypes;
-        revision-date 2019-07-01;
```

Comment:
leaf docker_routing_config_mode - default value should be separated, please check.

Reply:
This yang model is added by Praveen. He is the right person to modify this.

Reply:
Subject matter expert can raise PR to fix it.